### DISCUSSION TITLE
any option to filter out react videos? else feature request

### DISCUSSION THREAD

**Discussion by gdh123** [FreeTubeApp/FreeTube#4085] [Q&A]
hi folks,

I bloody hate react videos and some youtubers have 20% interesting videos the remaining 80 is just bloody react videos.
Any option I can remove them from my subscriptions page?

in case not, may I file a feature request?
free text field which contains xyz will not be displayed.
+ that could be useful also for parental control and is super generic (does not only apply to react videos)

**Comment by Mr-Bajs** [comment]
I don't think it's possible. 
The sponsorblock project does have the ability to tag whole videos such as Self Promotion or Exclusive Access for paid promotion. 
First i think Freetube would support to filter out videos that are tagged by sponsorblock as Self Promotional or Exclusive Access, then if that would be implemented then maybe if sponsorblock would add the option to tag videos as Reaction videos, then it would be possible i think. I doubt that Sponsorblock would implement this but i think thats your best option.

I think Sponsorblock is the best tagging and filtering solution as of now for Youtube. 

I would recomend to go over your channel subscriptions, to unsub from these channels that produce these low quality reaction videos.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Not possible because react content is not labeled maybe u could try to followup @Mr-Bajs suggestion? Also u dont have to file a report because there is already one https://github.com/FreeTubeApp/FreeTube/issues/3086
